### ISSUE TITLE
Feedback Moment Css

### ISSUE THREAD

**Issue by filippoboiani** [stagecast/bug-report#71]
**Feedback moment popup button**

**To Reproduce**
- launch feedback moment 
- open it 
- send feedback 
- the prompt button does not look right

**Expected behavior**
Bette visualisation

**Screenshots**
![feed](https://user-images.githubusercontent.com/17007951/48870267-68b5c400-ede0-11e8-8a3e-a27b10a1204d.jpg)

**Smartphone (please complete the following information):**
 - Device: iPhone6s
 - OS: 10.3.3

**Comment by peffis** [comment]
The button is also not clickable when I run it in a wkwebview. This might not be an issue for us yet, as I don’t know what webviews we run on the platforms at the moment, but this could be a sign that there is some html/css/js weakness in it that makes the code less portable. Anyway. Feel free to disregard this comment, but, if someone goes in and have a look at this button, perhaps check at the same time if there is something obviously “weak” when it comes to enabling/disabling this button

**Comment by fhyy** [comment]
I cannot reproduce this myself but is it the same for the Ideation moment? If not, I will compare with that code as they are almost identical.

**Comment by fhyy** [comment]
The not clickable issue is still unkown. I can't reproduce it and the button has only set the onclick function to act on this event. There is an asynchronous request made in the onclick function, maybe this causes some issues.

The button UI, I had fixed the width, so it did not handle slightly larger fonts properly.

**Comment by peffis** [comment]
Yes, the Ideation actually suffer from the same problem so something in that code makes it incompatible with wkwebview....which is not really a problem for us at the moment I think as we are not using wkwebview, but inconsistencies like that are always problematic and eventually we will switch over to wkwebview as well considering that has a much faster rendering/javascript engine. But don't worry about this now...but we make a mental note of it that the code in Feedback and Ideation is sensitive for some reason. And it seems like the button is not activated. I tap and tap on it but nothing happens.
